rllib/env/single_agent_env_runner.py

```diff
@@ -83,6 +91,9 @@ def __init__(self, config: AlgorithmConfig, **kwargs):
         except NotImplementedError:
             self.module = None
 
+        # Create a MetricsLogger object for logging custom stats.
+        self.metrics = MetricsLogger()
```

Comment:
Here as well: is this the right name? This is a class that does not only store but can also act